deploy/charts/cert-manager/templates/cainjector-deployment.yaml

```
@@ -151,7 +151,9 @@ spec:
       affinity:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- with .Values.cainjector.tolerations }}
+      {{- $tolerations := .Values.global.tolerations | default list }}
+      {{- $tolerations = concat $tolerations (.Values.cainjector.tolerations | default list) }}
+      {{- with $tolerations }}
```

```suggestion
      {{- with .Values.cainjector.tolerations | default .Values.global.tolerations }}
```

IMO it makes little sense to concat the global and local tolerations. Why did you choose to do this? A dict is quite different from an array.